DirectEmployers/MyJobs
# [PD-2732] replaced share-icon.png with share-icon.svg for better rende…

Purpose of PR:  Replace share-icon.png with share-icon.svg for faster rendering and image quality.

To test:

1. Please switch to v2 template.

2. Please ensure the share icon is there and not blurry.  Please open the network tab and ensure the share-icon.svg file isn't 1 trillion million yottabyte in size and takes 2 hours to render.
